### ISSUE TITLE
Tests fail on upcoming 2.7

### ISSUE THREAD

**Issue by GoogleCodeExporter** [testing-cabal/unittest-ext#15]
```
See https://bugs.gentoo.org/attachment.cgi?id=234239&action=view.

(This is with 0.4.1.)
```

Original issue reported on code.google.com by `djc.ochtman` on 6 Jun 2010 at 9:33

**Comment by GoogleCodeExporter** [comment]
```
Yep, the DeprecationWarning is silenced by default in Python 2.7. This will be 
fixed
in the next release. Thanks.
```

Original comment by `fuzzyman` on 6 Jun 2010 at 6:12
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.4.2.
```

Original comment by `fuzzyman` on 6 Jun 2010 at 7:06
- Changed state: **Fixed**
